package.json

```diff
@@ -8,7 +8,7 @@
     "@tensorflow/tfjs-backend-wasm": "^2.7.0",
     "@tensorflow/tfjs-core": "^2.7.0",
     "@tensorflow/tfjs-layers": "^2.8.0",
-    "danfojs": "^0.1.2",
+    "danfojs-node": "^0.1.10",
```

Comment:
datacook might be for the environment browser-compatible, thus this will be still danfojs?

Reply:
The Series object is the only feature I'm currently using from danfo, so I'm suggesting removing this for now since I can write it without using Series. Also, danfo (browser version) does not currently export a typescript declaration file, and as such doesn't compile.